Flat/interface/AnalyzerUtilities.h

```diff
@@ -112,6 +112,7 @@ class Analysis {
   bool useCMVA = false;
   bool varyJES = false;
   bool vbf = false;
+  bool applyMCTriggers = false;
```

Comment:
This is still here

Reply:
You told me to put it there, didn't you? It was in PandaAnalyzer, and it's now in Analysis. Am I wrong?

Reply:
Ah I got confused at what file I was looking at. My mistake, sorry!

Reply:
no prob! you are already doing a lot :-)